Wireframe/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  background-color: lavender;
+}
+
+.hero-image {
+  position: relative;
+  width: 100%;
+  height: 300px; /* Adjust height as needed */
+  overflow: hidden;
+}
+
+.hero-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover; /* Ensures the image covers the entire area */
+}
+
+.hero-text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  color: navy;
+}
+
+.text-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px;
+}
+
+main article {
+  flex: 1; /* Ensures each article takes equal space */
+  margin: 100px 10px;
+}
+
+
+h2 {
+  text-align: auto;
+  color: darkturquoise;
+  text-shadow: 5px 5px 5px dimgrey;, 0 0 25px blue, 0 0 5px orange;
+  margin-top: 16px; 
+  margin-bottom: 20px; 
+}
+
+
+
+.question {
+  font-size: 28px;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-weight: bold;
+  color: blue;
+  margin-top: 20px;
+  margin-bottom: 20px; 
+}
+
+.main {
+  
+  padding: 20px;
+}
+
+
+
+p {
+  font-size: 22px;
+  font-weight: bold;
+  margin-bottom: 20px; 
+}
+
+section {
+  margin: 10 auto;
+  background-color: rgb(168, 215, 234);
+}
+
+.footer {
```

Comment:
I don't see your footer in the preview -- why might this be?

Reply:
Not sure. When I open the HTML/Css on  preview live server it shows all inclusive footer. Do you know why can you not see on your side?